File: Python/1.py
import os
from typing import List

# def reverse_array(nums):
#     ans = []
#     for i in range(len(nums)-1, -1, -1):
#         ans.append(nums[i])
#     return ans

def reverse_array(nums):
    left = 0
    right = len(nums)-1

    while left < right:
        nums[left], nums[right] = nums[right], nums[left]
        left += 1
        right -= 1

    return nums

nums = list(map(int, input().split()))
invoke = reverse_array(nums)
print(invoke)

File: CPP/1.cpp
// Print an array in reverse order.

#include<iostream>
#include<bits/stdc++.h>

std::vector<int> reverse_array(const std::vector<int>& nums) {
    std::vector<int> rev;
    for(int i=nums.size()-1;i>=0;i--){
        rev.push_back(nums[i]);
    }
    return rev;
}

int main() {
    int n;
    std::cin>>n;
    std::vector<int>ans(n);
    for(int i=0;i<ans.size();i++) {
        std::cin>>ans[i];
    }
    std::vector<int> invoke = reverse_array(ans);
    for(int i=0;i<invoke.size();i++) {
        std::cout<<invoke[i]<< " ";
    }
    return 0;
}

